### PR TITLE
Update EISeg test version：Polygon operation

### PR DESCRIPTION
**Pull request by geoyee**
更新有关EISeg的角点编辑的测试版本

### CODE REVIEW

File: contrib/EISeg/eiseg/inference/predictor/__init__.py [hunk 1]
```diff
@@ -1,11 +1,23 @@
+from paddle.fluid.layers.rnn import dynamic_decode
 from .base import BasePredictor
```
Comment (haoyuying) [inline contrib/EISeg/eiseg/inference/predictor/__init__.py]: 这个地方和之前的保持一致吧，我之前把冗余代码删掉了，可以借鉴之前的版本

File: contrib/EISeg/eiseg/model/is_hrnet_model.py [hunk 2]
```diff
@@ -1,17 +1,25 @@
 import paddle.nn as nn
```
Comment (haoyuying) [inline contrib/EISeg/eiseg/model/is_hrnet_model.py:1]: 删除冗余代码

Comment (haoyuying) [inline contrib/EISeg/eiseg/model/is_hrnet_model.py:1]: 删除冗余代码

File: contrib/EISeg/eiseg/model/modifiers.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
```
Comment (haoyuying) [inline contrib/EISeg/eiseg/model/modifiers.py]: 删

File: contrib/EISeg/eiseg/util/manager.py [hunk 4]
```diff
@@ -18,56 +18,47 @@
 
 class ComponentManager:
     """
-        Implement a manager class to add the new component properly.
-        The component can be added as either class or function type.
+    Implement a manager class to add the new component properly.
```
Comment (haoyuying) [inline contrib/EISeg/eiseg/util/manager.py]: 没太看懂这个注释是什么意思。

File: contrib/EISeg/README.md [hunk 5]
```diff
@@ -49,7 +43,7 @@ eiseg
 ```
 即可运行软件。
 
-### Conda
+#### Conda
```
Comment (haoyuying) [inline contrib/EISeg/README.md]: 这个不是conda安装，只是虚拟了个环境，这可以删去，目前应该不支持conda 安装。

File: contrib/EISeg/README.md [hunk 6]
```diff
@@ -60,73 +54,76 @@ pip install eiseg
...
-| Ctrl+M                | 加载模型参数     |
+### 界面模块
+
+1. 可在`显示`中选择需要显示的界面模块，正常退出时将会记录界面模块的状态和位置，下次打开自动加载。
 
 # 开发者
 
 [Yuying Hao](https://github.com/haoyuying), [Yizhou Chen](https://github.com/geoyee), [Lin Han](https://github.com/linhandev/), [GT](https://github.com/GT-ZhangAcer), [Zhiliang Yu](https://github.com/yzl19940819)
+
+<!-- pip install 'git+https://github.com/openvinotoolkit/datumaro' -->
```
Comment (haoyuying) [inline contrib/EISeg/README.md]: 删掉可以

File: contrib/EISeg/eiseg/model/is_hrnet_model.py [hunk 7]
```diff
@@ -1,17 +1,25 @@
...
 
 class HRNetModel(ISModel):
-    def __init__(self, width=48, ocr_width=256, small=False, norm_layer=nn.BatchNorm2D, **kwargs):
+    def __init__(self, width=48, ocr_width=256, small=False, backbone_lr_mult=0.1,
+                 norm_layer=nn.BatchNorm2D, **kwargs):
         super().__init__(norm_layer=norm_layer, **kwargs)
 
         self.feature_extractor = HighResolutionNet(width=width, ocr_width=ocr_width, small=small,
                                                    num_classes=1, norm_layer=norm_layer)
+        self.feature_extractor.apply(LRMult(backbone_lr_mult))
```
Comment (haoyuying) [inline contrib/EISeg/eiseg/model/is_hrnet_model.py]: 删掉

File: contrib/EISeg/eiseg/model/is_hrnet_model.py [hunk 8]
```diff
@@ -1,17 +1,25 @@
...
+        if ocr_width > 0:
+            self.feature_extractor.ocr_distri_head.apply(LRMult(1.0))
```
Comment (haoyuying) [inline contrib/EISeg/eiseg/model/is_hrnet_model.py]: 删掉

File: contrib/EISeg/eiseg/model/modifiers.py [hunk 9]
```diff
@@ -0,0 +1,10 @@
...
+class LRMult(object):
```
Comment (haoyuying) [inline contrib/EISeg/eiseg/model/modifiers.py]: 删掉

File: contrib/EISeg/eiseg/util/vis.py [hunk 10]
```diff
@@ -134,6 +134,8 @@ def draw_with_blend_and_clicks(
         )
         result = result.astype(np.uint8)
 
+        # result = (result * (1 - alpha) + alpha * rgb_mask).astype(np.uint8)
```
Comment (haoyuying) [inline contrib/EISeg/eiseg/util/vis.py]: 删掉

File: contrib/EISeg/eiseg/util/manager.py (hunk 4 above)
Comment (geoyee) [reply contrib/EISeg/eiseg/util/manager.py]: 这个应该是直接在seg下面的manager来的，我直接删掉吧

**Review by nepeplwu — APPROVED** [review]
